includes/class-wcpdf-settings-ubl.php

```diff
@@ -0,0 +1,145 @@
+<?php
+namespace WPO\WC\PDF_Invoices;
+
+use \WPO\WC\UBL\Settings\TaxesSettings;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+if ( ! class_exists( '\\WPO\\WC\\PDF_Invoices\\Settings_UBL' ) ) :
+
+class Settings_UBL {
+	
+	public $sections;
+
+	function __construct()	{
+		$this->sections = [
+			'taxes' => __( 'Taxes classification', 'woocommerce-pdf-invoices-packing-slips' ),
+		];
+		
+		add_action( 'admin_init', array( $this, 'init_tax_settings' ) );
+		add_action( 'wpo_wcpdf_settings_output_ubl', array( $this, 'output' ), 10, 1 );
+		
+		add_action( 'woocommerce_order_after_calculate_totals', array( $this, 'save_taxes_on_order_totals' ), 10, 2 );
+		add_action( 'woocommerce_checkout_order_processed', array( $this, 'save_taxes_on_checkout' ), 10, 3 );
+	}
+
+	public function output( $active_section ) {
+		$active_section = ! empty( $active_section ) ? $active_section : 'taxes';
+		?>
+		<div class="wcpdf_ubl_settings_sections">
+			<?php if ( count( $this->sections ) > 1 ) : ?>
+			<h2 class="nav-tab-wrapper">
+				<?php
+					foreach ( $this->sections as $section => $title ) {
+						$active = ( $section == $active_section ) ? 'nav-tab-active' : '';
+						printf( '<a href="%1$s" class="nav-tab nav-tab-%2$s %3$s">%4$s</a>', esc_url( add_query_arg( 'section', $section ) ), esc_attr( $section ), $active, esc_html( $title ) );
+					}
+				?>
+			</h2>
+			<?php else : ?>
+			<h3><?php echo $this->sections[$active_section]; ?></h3>
+			<?php endif; ?>
+		</div>
+		<?php
+		
+		switch ( $active_section ) {
```

Comment:
I know, is just prepare to be expanded. You can have different sections here, of course at this moment we just have 1 because I have removed the other and placed in the document settings.